cmd/cluster-operator/README.md

```diff
@@ -24,17 +23,16 @@ helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRD
 
 ```
 # must run this after cert-manager is ready
-kubectl create namespace kurator-system
-helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
-helm install kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz -n kurator-system
-helm install -f /root/capa-values.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
+helm install --create-namespace kurator-cluster-operator out/charts/cluster-operator-0.3-dev.tgz -n kurator-system
```

Comment:
can we put 0.3v-dev a into  var

Reply:
this's a user face doc, so I think it's OK.